.github/workflows/release-operator.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release Operator
+
+on:
+  pull_request:
+    types:
+    - closed
+    branches:
+      - 'release-v[0-9]+.[0-9]+'
+  workflow_dispatch: {}
+
+jobs:
+  if_merged:
```

Comment:
job name should be `release` or `release-operator`, `if_merged` is more a state, condition, etc than a job name.